src/main/webapp/location-form.html

```diff
@@ -3,14 +3,41 @@
 <head>
     <meta charset="UTF-8">
     <title>location-form</title>
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 </head>
 <body>
 
 <div class="padded rounded panel centered">
     <h1 id="page-title">Location Form</h1>
+    <br>
+    <br>
 </div>
 
-<div class="padded rounded panel">
+<form id="location-form" class="hidden" action = "/location" method="POST">
+    <h3>Enter a new Location:</h3>
+    <div class="form-group">
+      <label for="location-name">Name your Location</label>
+      <input type="name" class="form-control" id="location-name" aria-describedby="locationHelp" placeholder="Location Name">
+    </div>
+    <div class="form-group">
+      <label for="locationDescription">Enter a description</label>
+      <input type="description" class="form-control" id="locationDescription" placeholder="Location Description">
+    </div>
+    Add your location to the  
+        <a href="user-map.html" target="_blank">Map</a> 
```

Comment:
This pops up a page with user-map.html. If the user makes markers there, are they linked somehow to this location?